tests/build-cluster-dns.yaml

```diff
@@ -0,0 +1,36 @@
+test_duration: 5760
```

Comment:
1. Change the test duration to 15m.
2. Change the name of the YAML to dns-cluster-15min.yaml